frontend/src/components/ProjectList/components/Dataset/components/DownloadDataset/components/Content/components/CurlLink/style.ts

```diff
@@ -0,0 +1,45 @@
+import { GREY } from "src/components/common/theme";
+import styled from "styled-components";
+
+export const CodeWrapper = styled.div`
+  position: relative;
+  width: 511px;
+`;
+
+export const Code = styled.code`
+  border: 1px solid #cdcdcd;
```

Comment:
Might want to standardize this like BLUE at some point?

Reply:
Ooh thanks for catching this, Pat! Yes we have enum for `GREY` already, so I just added this one as `VERY_LIGHT`, according to https://www.htmlcsscolor.com/hex/CDCDCD#:~:text=%23CDCDCD%20(or%200xCDCDCD)%20is,of%20max%20value%20%3D%20765 😄 

Thank you!